### PR TITLE
fuel computer in SD logs

### PR DESCRIPTION
**Pull request by mck1117**
#4727 #4826

Gets this stuff back in SD logs.

### CODE REVIEW

File: firmware/controllers/math/speed_density.cpp [hunk 1]
```diff
@@ -120,5 +119,4 @@ temperature_t IFuelComputer::getTCharge(int rpm, float tps) {
 void initSpeedDensity() {
 	veMap.init(config->veTable, config->veLoadBins, config->veRpmBins);
 //	ve2Map.init(engineConfiguration->ve2Table, engineConfiguration->ve2LoadBins, engineConfiguration->ve2RpmBins);
-	lambdaMap.init(config->lambdaTable, config->lambdaLoadBins, config->lambdaRpmBins);
```
Comment (rusefillc) [inline firmware/controllers/math/speed_density.cpp:123]: how is this related to "constexpr fue computer"?

File: firmware/console/binary/output_channels.txt [hunk 2]
```diff
@@ -74,7 +74,6 @@ uint16_t rpmAcceleration;dRPM;"RPM/s",1, 0, 0, 5, 2
 	uint16_t autoscale chargeAirMass;@@GAUGE_NAME_AIR_MASS@@;"g",{1/1000}, 0, 0, 0, 3
 	uint16_t autoscale crankingFuelMs;crankingFuelMs\nairmass in mg, 0-65 grams;"ms",{1/@@PACK_MULT_FUEL_MASS@@}, 0, 0, 0, 2
 
-	uint16_t autoscale currentTargetAfr;@@GAUGE_NAME_TARGET_AFR@@;"ratio",{1/@@PACK_MULT_AFR@@}, 0, 0, 0, 2
```
Comment (rusefillc) [inline firmware/console/binary/output_channels.txt]: this is not part of "constexpr fuel computer"

File: firmware/controllers/math/speed_density.cpp (hunk 1 above)
Comment (mck1117) [reply firmware/controllers/math/speed_density.cpp:123]: It changes the target AFR table to match how the other stuff like it works (advance map, injection timing, trim, etc)

File: firmware/console/binary/output_channels.txt (hunk 2 above)
Comment (mck1117) [reply firmware/console/binary/output_channels.txt]: That is not in the total diff, are you looking at a single commit?

Comment (rusefillc) [reply firmware/console/binary/output_channels.txt]: yes, i look at single commits to divide and conquer

File: firmware/controllers/math/speed_density.cpp (hunk 1 above)
Comment (rusefillc) [reply firmware/controllers/math/speed_density.cpp:123]: please extract this change into a separate PR

File: firmware/console/binary/output_channels.txt (hunk 2 above)
Comment (mck1117) [reply firmware/console/binary/output_channels.txt]: that commit is from a previous PR, I branched before it merged.

File: firmware/controllers/math/speed_density.cpp (hunk 1 above)
Comment (mck1117) [reply firmware/controllers/math/speed_density.cpp:123]: will make an attempt in 6-8 hours

**Comment by rusefillc** [comment]
With TS bug and SD lack of logging now being separate tickets I believe that this PR is referencing wrong tickets

**Comment by rusefillc** [comment]
@mck1117 please rebase (_rebase_)

**Comment by rusefillc** [comment]
@mck1117 any arguments against rebase?

**Comment by mck1117** [comment]
> @mck1117 any arguments against rebase?

I have been on an airplane or asleep since your first request
